src/Swarm/TUI/Controller.hs

```diff
@@ -507,6 +509,10 @@ handleInfoPanelEvent s (VtyEvent (V.EvKey V.KEnter [])) = do
               & gameState . replStatus .~ REPLWorking mkTy Nothing
               & gameState . robotMap . ix "base" . machine .~ initMachine mkPT topEnv
         _ -> continueWithoutRedraw s
+
+handleInfoPanelEvent s (VtyEvent (V.EvKey (V.KChar '/') []))
+  = continue $ s & uiState . uiSearching .~ True
```

Comment:
nit: there is no feedback when entering search mode, should we display an indicator indicating the ui is waiting for input? Also perhaps we should add the key to the help bar, e.g. next to the navigate tooltip `[/] search`.

Reply:
This bothered me as well.  I'll have to dig in to `brick` more to figure out how to do what I want which is blinking cursor on search `Separator`.  It of course doesn't have to be a separator, but that already existed.  It also has been bothering me that the cursor continues to blink in the repl pane when focus is in another pane.